b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeSwitchOrg.js

```diff
@@ -32,7 +33,7 @@ export default async function decodeSwitchOrg(request, token) {
     const accessToken = token.accessToken;
 
     try {
-        const orgSession = await callSwitchOrg(subOrgId, accessToken);
+        const orgSession = await commonDecode(()=> callSwitchOrg(subOrgId, accessToken), null);
```

Comment:
formatting issues. Leave a space before and after =>. Check other places also

Reply:
Fixed when this functionality moved to a new file

### Commit
- https://github.com/Achintha444/samples-is/commit/47a6eb49f960af918ac35a40ee108f27160064ed#diff-1c2ff29757872db35612bab40a6e48d0cdb3bbc52f8b75597704ddf30dc6c114